pynsee/sirene/get_location.py

```diff
@@ -13,17 +13,23 @@
 from pynsee.geodata.GeoFrDataFrame import GeoFrDataFrame
 from pynsee.sirene._get_location_openstreetmap import _get_location_openstreetmap
 
+import logging
+logger = logging.getLogger(__name__)
 
 @lru_cache(maxsize=None)
 def _warning_get_location():
-    print(
-        "For at least one point, exact location has not been found, city location has been given instead"
+    logger.warning(
+        "For at least one point, exact location has not been found, city "
+        "location has been given instead"
     )
 
 @lru_cache(maxsize=None)
 def _warning_OSM():
-    print("This function returns data made available by OpenStreetMap and its contributors")
-    print("Please comply with Openstreetmap's Copyright and ODbL Licence")
+    logger.info(
```

Comment:
must be warning

Reply:
This is something that should be stated in documentation (and maybe a warning in the README), not an unexpected behaviour.

Reply:
fair enough, I'll open an issue about that